lib/assessment.rb

```diff
@@ -6,13 +6,23 @@
 # Make all of those attributes read-only.
 
 ##
-# your code here
+class Dog
+  attr_reader: name
+  attr_reader: breed
+  attr_reader: age
+
+  def initialize(name,breed,age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
 ##
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+Dog.new = ("Sluggo", "Pitbull", 2) # replace nil with your code
```

Comment:
This doesn't really create a new object. Hint: you shouldn't have changed `Answer.dog`.